SampleUsages/TestSamples/Http-CSharp-CPUIntensive/azuredeploy.json

```diff
@@ -0,0 +1,105 @@
+            },
+            "config": {
+              "bindings": [
+                {
+                  "type": "httpTrigger",
+                  "name": "req",
+                  "authLevel": "function",
+                  "direction": "in"
+                }
+              ] 
+            }
+          }
+        }
+      ],
+      "type": "Microsoft.Web/sites"
+    }
+  ],
+    "variables": {
+        "functionAppName": "[concat('http-csharp-cpu-intensive', uniquestring(resourceGroup().id))]",
```

Comment:
same as node sample, except for name change in functionAppName & hostingPlanName